package-lock.json

```diff
@@ -1,27 +1,31 @@
 {
   "name": "pokemon-showdown",
-  "version": "0.11.0",
```

Comment:
I don't recall seeing this change. I though doing ``npm uninstall emoji-regex`` would have reverted the changes from the initial commit which affected package-lock.json